src/components/operatorsList.tsx

```diff
@@ -82,6 +91,13 @@ export const OperatorsList: React.FC<OperatorsListProps> = ({ operatorOwner }) =
                   <Td {...textStyles.text} isNumeric>
                     {hexToFormattedNumber(operator.operatorDetail.currentTotalStake)}
                   </Td>
+                  {isOneOfTheOperators &&
```

Comment:
I guess something here might not be matching in my case, hence I don't see any available actions

Reply:
Thanks I believe I fix this in my last push